src/frontend/context/TogglesContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react';
+
+import createUseContext from 'constate';
+import { Toggles } from './toggles';
+
+const [TogglesProvider, useToggles] = createUseContext(() => {
+    const [toggles, settToggles] = useState<Toggles>({});
+    TogglesProvider.displayName = 'TOGGLES_PROVIDER';
+
```

Comment:
Tror det er lurt hvis du henter toggles her i stil med noe sånt. 

`import { useCallback, useEffect, useState } from 'react';

import createUseContext from 'constate';
import { Toggles } from './toggles';
import axios, { AxiosResponse } from 'axios';

const [TogglesProvider, useToggles] = createUseContext(() => {
    const [toggles, settToggles] = useState<Toggles>({});
    TogglesProvider.displayName = 'TOGGLES_PROVIDER';

    const fetchToggles = useCallback(() => {
        const hentToggles = () => {
            return axios.get(`/familie-ef-sak/api/featuretoggle`, {
                withCredentials: true,
            });
        };

        hentToggles()
            .then((resp: AxiosResponse<Toggles>) => settToggles(resp.data))
            .catch((err: Error) => {
                console.log('Kunne ikke hente toggles, ' + err.message);
            });
    }, []);

    useEffect(() => {
        fetchToggles();
    }, [fetchToggles]);

    return { toggles };
});

export { TogglesProvider, useToggles };`

Reply:
Ja, det ser mye mere fornuftig ut. Kunne ha gjort det samme i ef-soknad og ?

Reply:
Kanske vi borde ha bedre feilhåndtering eller håller det å logga det consolen ?

Reply:
Ja, det hjelper jo ikke så mye å logge til konsollen. Men hvordan feilhåndtere her ?

Reply:
Vi har jo denne på serversiden;
```

    router.post('/logg-feil', (req: Request, res: Response) => {
        logRequest(req, req.body.melding, LOG_LEVEL.ERROR);
        res.status(200).send();
    })
    
    ```
    
    Kanskje noe vi burde ta i bruk for logging av feilsituasjoner?

Reply:
Resolver denne (Ref diskusjon med Charlie der vi ble enige om å bare logge foreløpig)